_data/people.yml

```
@@ -152,6 +152,21 @@
         icon: envelope-o
         name: email
 
+- name: Amanda Bachmann
+  pic: /img/people/BachmannA.png
```

For consistency with the other files*, can you please make the filename all lowercase? The file will also have to be changed with `git mv BachmannA.png bachmanna.png"

```suggestion
  pic: /img/people/bachmanna.png
```

* Also, in general, I am lazy and I prefer filenames to have no capital letters, when possible.